tools/buildutils/installbazel.sh

```diff
@@ -19,13 +19,17 @@
 set -e
 
 function install_bazel_x86_64() {
-  echo "Installing bazel"
-  apt install apt-transport-https curl gnupg -y
-  curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
-  mv bazel-archive-keyring.gpg /usr/share/keyrings
-  echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
-  # bazel needs the zip command to gather test outputs but doesn't depend on it
-  apt-get update && apt-get install -y bazel zip unzip
+  if command -v bazel &> /dev/null; then
```

Comment:
let's put this condition around `sudo "${INSTALL_BAZEL}"` in https://github.com/google/android-cuttlefish/blob/main/tools/buildutils/build_packages.sh#L28

Reply:
done. re-commit.